dlk/python/dlk/core/view.py

```diff
@@ -415,31 +415,6 @@ def run(self):
                 """
             )
 
-            if len(input_ops) != 2:
-                self.raise_invalid_args_exception(op, input_ops, output_ops)
-
-            main_op = input_ops[0]
-            sub_op = input_ops[1]
-
```

Comment:
So `Sub` was magically turning into `Mul` sometimes?

Reply:
I think either forgot to insert conditional `elif` for the `Mul` or it was duplicated condition...